volttrontesting/fixtures/docker_wrapper.py

```diff
@@ -47,29 +51,44 @@ def create_container(image_name: str, ports: dict = None, env: dict = None, comm
 
         # Create docker client (Uses localhost as agent connection.
         client = docker.from_env()
-        if ":" in image_name:
-            client.images.pull(image_name)
-        else:
-            # So all tags aren't pulled. According to docs https://docker-py.readthedocs.io/en/stable/images.html.
-            client.images.pull(image_name + ":latest")
-        container = client.containers.run(image_name, ports=ports, environment=env, auto_remove=True, detach=True)
+
+        try:
+            repo = image_name
+            if ":" not in repo:
+                # So all tags aren't pulled. According to docs https://docker-py.readthedocs.io/en/stable/images.html.
```

Comment:
I don't think it would pull all of the tags...we just are being explicit about pulling the latest tag.  unless somewhere on the linked page tells me otherwise....in which case you should like to the exact location in that page.

Reply:
I tested out the call without ":" on the interpreter. It looks like it does pull all the images if a tag is not given. I had to kill the process since it was taking too long. Then I checked docker for images and a bunch of images appeared.
```bash
>>> import docker
>>> client = docker.from_env()
>>> image = "mysql"
>>> res = client.images.pull(image)
^CTraceback (most recent call last):

# Checking docker
$ docker images ls

REPOSITORY                                                     TAG                 IMAGE ID            CREATED             SIZE
crate                                                          latest              26e2f7f0075c        13 days ago         676MB
mysql                                                          5.7                 9cfcce23593a        2 weeks ago         448MB
mysql                                                          latest              be0dbf01a0f3        2 weeks ago         541MB
670892884287.dkr.ecr.us-gov-west-1.amazonaws.com/lessons/api   0.2.9               f24c01f59ca8        6 months ago        594MB
670892884287.dkr.ecr.us-gov-west-1.amazonaws.com/lessons/api   latest              f24c01f59ca8        6 months ago        594MB
mysql                                                          5.5.57              6929c9b0e536        2 years ago         257MB
mysql                                                          5.5.56              17e9a1fe381e        3 years ago         256MB
mysql                                                          5.5.55              c0fb485deb70        3 years ago         256MB
mysql                                                          5.5.54              9913483658d3        3 years ago         256MB
mysql                                                          5.5.53              0e21a45a5660        3 years ago         255MB
mysql                                                          5.5.52              402df65b897e        3 years ago         255MB
mysql                                                          5.5.51              51ae120c924c        3 years ago         256MB
mysql                                                          5.5.50              385c5c57528f        3 years ago         256MB
...........and more images



```